packages/react-ui-components/src/Icon/style.css

```diff
@@ -20,10 +20,10 @@
     font-size: .5em;
 }
 .icon--paddedLeft {
```

Comment:
Should we adjust the classname as well?

Reply:
We talked about that with @dimaip. He said the word "padded" just means the icon has a space next to it and does not correspond with the css-property "padding". So according to him it does not matter if we use css-margin or css-padding to realise what this classname means.